packages/app-store/zohocrm/api/_getAdd.ts

```diff
@@ -0,0 +1,20 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { WEBAPP_URL } from "@calcom/lib/constants";
+
+import getAppKeysFromSlug from "../../_utils/getAppKeysFromSlug";
+
+let client_id = "";
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (req.method !== "GET") return res.status(405).json({ message: "Method not allowed" });
```

Comment:
Don't think you need to do it if you are using defaultHandler. There is this logic here https://github.com/calcom/cal.com/blob/87dbfba5dfab129eb74fa4973504f8dfd5571e4c/packages/lib/server/defaultHandler.ts#L11

Reply:
Ah good call @hariombalhara ! Fixed it. Thanks